### PR TITLE
style: remove extra space at top-pages

### PR DESCRIPTION
**Pull request by BinaHovav**
hey i removed extra space in 3 pages:

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "underscore.string": "^3.3.6",
     "use-konami": "^1.0.1",
     "usehooks-ts": "^2.9.1",
-    "vite": "^4.4.9",
```
Comment (shootermv) [inline package.json:40]: it is good to upgrade vite package to 5 version
but this is something to do with different PR (in my opinion)
we better do only changes solves the problem which PR tries to solve (the issue of PR)

File: src/layout/index.tsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ const StyledContent = styled(Content)`
 `
 const StyledBody = styled.div`
   padding: 24px;
+  padding-top: 0;
```
Comment (shootermv) [inline src/layout/index.tsx:23]: you can write it shorter: `padding: 0 24px `

File: package.json (hunk 1 above)
Comment (BinaHovav) [reply package.json:40]: this is weird, i don't remember touching the vite package version

File: src/layout/index.tsx (hunk 2 above)
Comment (BinaHovav) [reply src/layout/index.tsx:23]: you're right

File: package.json (hunk 1 above)
Comment (NoamGaash) [reply package.json:40]: Next time, make sure not to stage everything in your working directory - adding individual files is a good practice :)

File: src/layout/index.tsx (hunk 2 above)
Comment (NoamGaash) [inline src/layout/index.tsx:23]: @shootermv this comment can be [a suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) :)
```suggestion
  padding: 0 24px;
```

File: yarn.lock
Comment (NoamGaash) [inline yarn.lock:1]: please delete this file - it's not needed now as we moved to NPM

**Review by NoamGaash — CHANGES_REQUESTED** [review]
Thanks for your contribution!
Once the `yarn.lock` will be deleted, I'll be more than happy to approve this pull request.
Welcome!

File: package.json (hunk 1 above)
Comment (NoamGaash) [reply package.json:40]: the `npm ci` command is failing - it means the `package` and `package-lock` files are not synced.
There are two valid solutions:
1. revert `package.json`
2. run `npm install` and commit the changes in the `package-lock.json` file.

Comment (BinaHovav) [reply package.json]: Thanks to @shootermv now it's all fixed :)

File: yarn.lock
Comment (BinaHovav) [reply yarn.lock:1]: Done

**Review by NoamGaash — APPROVED** [review]
Looks great! congrats for your first contribution :balloon:
